CHANGELOG.md

```diff
@@ -41,6 +41,17 @@ All notable changes to this project will be documented in this file.
 - `0.2.x` Releases - [0.2.0](#020)
 - `0.1.x` Releases - [0.1.0](#010)
 
+## [5.0.1](https://github.com/Nike-Inc/SQift/releases/tag/5.0.1)
+
+Release on 2019-06-13.  All issues associated with this milestone can be found using this
+[filter](https://github.com/Nike-Inc/SQift/milestone/12?closed=1).
+
+#### Updated
+
+-  Fixes Carthage build issues with failing tests. Updates to Scheme's allows carthage to do the build.
```

Comment:
How about?

```
- Fixed issue where Carthage was unable to build SQift due to schemes requiring test suite compilation.
  - Fixed by [William]...
```

Reply:
Updated.

Reply:
This is still not quite right. Both should start with `Fixed` and typo in `was`. Second line is indented two spaces.

Reply:
Done